src/templates/instructions.html

```diff
@@ -18,7 +18,16 @@
         <% steps.forEach(function(step) { %>
           <%
             var distance = step.distance ? format(step.distance) : false;
-            var icon = step.maneuver.type.replace(/\s+/g, '-').toLowerCase();
+            var icon = step.maneuver.modifier ? (step.maneuver.modifier).replace(/\s+/g, '-').toLowerCase() : step.maneuver.type.replace(/\s+/g, '-').toLowerCase();
+
+            if (step.maneuver.type==="arrive" || step.maneuver.type === "depart"){
```

Comment:
Nitpicky but can you use single quotes and a space between the closing parenthesis and opening bracket?